src/SFML/Window/OSX/SFViewController.mm

```diff
@@ -172,6 +172,18 @@ - (void)resizeTo:(unsigned int)width by:(unsigned int)height
 }
 
 
+////////////////////////////////////////////////////////
+- (void)setMinimumSize:(NSSize)size
+{
+}
+
+
+////////////////////////////////////////////////////////
+- (void)setMaximumSize:(NSSize)size
+{
+}
+
+
```

Comment:
I wasn't entirely sure what this file did. Does it make sense to set the min or max size on a view?

Reply:
I don't think so.